### PR TITLE
Algorithm for Singly Linked List.

### PR DESCRIPTION
**Pull request by Anishukla**

### CODE REVIEW

File: SinglyLinkedList/SinglyLinkedList.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        if c == 0:        
+            print("The value isn't present.")
+
+mylist = LinkedList()
+print("Choose 1.To append at start, 2.To append in between, 3.To append at end." )
+print("4.To delete from start, 5.To delete from between, 6. To delete from end.")
+print("7.To View Linkedlist, 8. Search a value's position, 9.To view and exit.")
+
+while(1):
+    K = int(input())
```
Comment (starkblaze01) [inline SinglyLinkedList/SinglyLinkedList.py]: Hi, thanks for the contribution @Anishukla. Can you do a small change? When we are taking `K` input can print there
```
K = int(input('choose an option between 1 to 9')) 
```

**Comment by Anishukla** [comment]
Okay, I'll do that.

**Comment by Anishukla** [comment]
I've made the changes. You can check it now!

**Comment by Anishukla** [comment]
Your welcome! Will try and add some more algorithms soon!

**Pull request merged**